app/build.gradle

```diff
@@ -157,7 +157,7 @@ afterEvaluate {
     project.tasks.matching { it.name.startsWith('jaxRs2Retrofit') }.each {
         it.inputDir = file(project(':../fablab-common').projectDir.toString() + '/src/main/java')
         it.outputDir = generatedSourcesDir
-        it.packageName = 'de.fau.cs.mad.fablab.rest.myapi'
+        it.packageName = 'de.fau.fablab.app.rest.myapi'
```

Comment:
Ah, I reread the complete code. In the excerpt I misstook it for changing the packagename.
But I think this should be OK.